XRTK-Core/Packages/com.xrtk.core/Runtime/Utilities/CameraCache.cs

```diff
@@ -52,5 +52,21 @@ public static Camera Refresh(Camera newMain)
 
             return cachedCamera = newMain;
         }
+
+        /// <summary>
+        /// Validates if the CameraCache reference is Null to avoid creating a new camera if it does not exist.
+        /// </summary>
+        public static bool IsNull
```

Comment:
Love this change

Reply:
Here's hoping that is truthful and not mocking :D